statistics/statistics.go

```diff
@@ -212,94 +276,135 @@ func (c *Column) mergeBuckets(bucketIdx int64) {
 	return
 }
 
-// columnToPB converts Column to ColumnPB.
-func columnToPB(col *Column) (*ColumnPB, error) {
-	data, err := codec.EncodeValue(nil, col.Values...)
+// Table represents statistics for a table.
+type Table struct {
+	Info    *model.TableInfo
+	Columns []*Column
+	Indices []*Column
```

Comment:
@lamxTyler 
We can extract the same logic into a type like `histogram`, then wrap it with `Column` and `Index`.